app/helpers/strong_parameters_helper.rb

```diff
@@ -0,0 +1,98 @@
+module StrongParametersHelper
+  def permitted_attributes
+    Foreman::PermittedAttributes
+  end
+
+  def foreman_params
```

Comment:
Why is this method ```foreman_params``` instead of ```permitted_params```? It is not very descriptive unfortunately as it is currently.